src/conditional-flow/string-conditions.js

```diff
@@ -1,41 +1,73 @@
 // This function should return true if the passed string is equal to "Hello"
-function isHello (val1) {
-
+function isHello(val1) {
+  if (val1 === "Hello") {
+    return true;
+  } else {
+    return false;
+  }
   // TODO: write code in this function body to pass the tests
-
 }
 
 // This function should return true if the passed string is not equal to "Hello"
-function isNotHello (val1) {
-
+function isNotHello(val1) {
+  if (val1 !== "Hello") {
```

Comment:
Same as above, the if statement here isn't required as `val1 !== "Hello"` is a boolean statement.